src/Test/IntegrationTestCase.php

```diff
@@ -32,7 +32,7 @@ abstract class IntegrationTestCase extends TestCase
     /**
      * @return string
      */
-    abstract protected function getFixturesDir();
+    abstract protected static function getFixturesDir();
```

Comment:
This change is a BC break. It cannot be done in a minor version as is.

Reply:
Correct. I already mentioned it in the issue and in this PR. I was told to open a PR against 3.x. Can you guide me to a better solution?

Reply:
What do you think of providing a second `IntegrationTestCase` which supports PHPUnit10 and deprecating the old one? If you agree, do you have suggestions for a name?

Reply:
But do you need to fix it? Non-static data providers are only deprecated but should still work in PHPUnit 10 right?

Reply:
they are deprecated in 9.6 and unsupported in 10

Reply:
Well, then I must have misunderstood this line: https://github.com/sebastianbergmann/phpunit/blob/main/ChangeLog-10.0.md?plain=1#L99

Reply:
ah indeed. I confused with the Test suffix for abstract classes.

Reply:
They are still working in PHPUnit 10. It show a deprecations notice at the end. 
So yes, this would be unsupported in PHPUnit 11.